plugins/modules/ec2_vpc_route_table.py

```diff
@@ -283,10 +278,10 @@ def find_subnets(connection, module, vpc_id, identified_subnets):
     """
     Finds a list of subnets, each identified either by a raw ID, a unique
     'Name' tag, or a CIDR such as 10.0.0.0/8.
-
-    Note that this function is duplicated in other ec2 modules, and should
-    potentially be moved into a shared module_utils
     """
+    CIDR_RE = re.compile(r'^(\d{1,3}\.){3}\d{1,3}/\d{1,2}$')
+    SUBNET_RE = re.compile(r'^subnet-[A-z0-9]+$')
+
```

Comment:
Yes another candidate for Py3's `ipaddress` module :-).

Reply:
Yeah I went looking for something existing we could reuse, but as far as I found nothing is there today